content/user/troubleshooting/debug_logs.md

```diff
@@ -88,6 +88,7 @@ Some logs will follow special file naming conventions to make them more easily i
 
 - `myDEBUG-bounced-email-adm-` failure to send email from admin
 - `myDEBUG-bounced-email-` failure to send email from storefront
+- `zcInstallLog_` - log created by the `zc_install` process.
```

Comment:
Two notes:
- `zcInstallLog_*` files are auto-cleaned up / pruned upon logging into the Admin console, so are short-lived
- `zcDEBUG-*` files report fatal errors which occurred during the zc_install process (and are not auto-pruned)